<?xml version="1.0"?>
<launch>
    <arg name="config_file" default="$(find arena_camera)/config/arena_default.yaml" />
    <!--arena camera-->
    <node name="arena_camera_node" pkg="arena_camera" type="arena_camera_node" output="screen"
        respawn="false" launch-prefix="">
        <rosparam command="load" file="$(arg config_file)" />
    </node>
    <!--AHRS IMU-->
    <node name="ahrs_driver_node" pkg="ahrs_driver" type="ahrs_driver_node" output="screen" 
        respawn="true">
    <!--image_view-->
    </node>
    <node name="image_view" pkg="image_view" type="image_view" output="screen" respawn="false">
    <remap from="image" to="/arena_camera_node/image_raw" />
        <param name="autosize" value="false" />
    </node>
</launch>